torch/jit/frontend.py

```diff
@@ -178,6 +178,34 @@ def get_jit_class_def(cls, self_name):
     ctx = SourceContext(source, filename, file_lineno, leading_whitespace_len, False)
     return build_class_def(ctx, py_ast.body[0], methods, properties, self_name)
 
+def check_and_indent_multiline_strings(sourcelines):
+    """
+    This is a helper function which checks for multiline strings and
+    indents the strings by calculating the leading space and appending
+    the spaces to each line of the multiline string.
+
+    Arguments:
+        sourcelines: This is an array of source lines of the function
+    Returns:
+        This function returns the updated indented sources,i.e,sourcelines
+
+    """
+    indices = []
+    # Extract the start and end line number of the multiline string
+    for index, source in enumerate(sourcelines):
+        if '\"\"\"' in source and source.find('\"\"\"') == source.rfind('\"\"\"'):
+            indices.append(index)
+    # If no multiline string, we have nothing to do.
+    if len(indices) == 0:
```

Comment:
i guess this block helps readability, although it is technically not needed since your for loop will not execute for len=0

Reply:
Yeah, it is redundant. Thanks for pointing this. Have removed it